### ISSUE TITLE
always use object shorthand

### ISSUE THREAD

**Issue by mishelashala** [ksquarelabsmx/react-guidelines#3]
Title: favor object shorthand syntax
Date: 27/02/2019
By: Michell Ayala

## Motivations
From eslint: ECMAScript 6 provides a concise form for defining object literal methods and properties. This syntax can make defining complex object literals much cleaner.

## Examples

Variables and properties with the same names
```js
// without object shorthand
const name = 'Juan Perez'
const user = {
  name: name
}

// with object shorthand
const name = 'Juan Perez'
const user = {
  name
}
```

## Sources
[ESlint docs](https://eslint.org/docs/rules/object-shorthand)

**Comment by mishelashala** [comment]
@Stidro @EduardoMY @Rodmg what do you think guys?

**Comment by Rodmg** [comment]
I approve 👍

**Comment by basultobdKs** [comment]
Definitely yes!

**Comment by mishelashala** [comment]
It was merged on #6
